packages/devtools_app/lib/src/tree.dart

```diff
@@ -188,15 +192,12 @@ class _TreeViewItemState<T extends TreeNode<T>> extends State<TreeViewItem<T>>
   }
 
   void _onExpanded() {
-    if (widget.onItemExpanded == null) {
-      return;
-    }
-    widget.onItemExpanded(widget.data);
+    widget?.onItemExpanded(widget.data);
     setExpanded(widget.data.isExpanded);
   }
 
   void _onSelected() {
-    widget.onItemSelected(widget.data);
+    widget?.onItemSelected(widget.data);
     setExpanded(widget.data.isExpanded);
```

Comment:
should we be calling onItemExpanded here as well?

Reply:
No, since `onItemExpanded` corresponds to clicking the `>`, whereas `onItemSelected` corresponds to clicking on any other part of the list item.